common/zoom/zoom-scheduled-meeting.ts

```diff
@@ -12,41 +12,36 @@ enum RecurrenceMeetingTypes {
     MONTHLY = 3,
 }
 
-type ZoomMeeting = {
+export type ZoomMeeting = {
+    agenda: string;
+    created_at: string;
+    duration: number;
+    host_id: string;
+    id: number;
+    join_url: string;
+    pmi: string;
+    start_time: string;
+    timezone: string;
+    topic: string;
+    type: number;
+    uuid: string;
+};
+
+export type ZoomMeetings = {
     next_page_token: string;
     page_count: number;
     page_number: number;
     page_size: number;
     total_records: number;
-    meetings: [
-        {
-            agenda: string;
-            created_at: string;
-            duration: number;
-            host_id: string;
-            id: number;
-            join_url: string;
-            pmi: string;
-            start_time: string;
-            timezone: string;
-            topic: string;
-            type: number;
-            uuid: string;
-        }
-    ];
+    meetings: ZoomMeeting[];
```

Comment:
I'm pretty sure this returns an array of ZoomMeetings, not a Tuple, right? (ZoomMeeting[] instead of [ZoomMeeting])